internal/provider/kubernetes/predicates.go

```diff
@@ -76,29 +76,18 @@ func (r *gatewayAPIReconciler) checkObjectNamespaceLabels(nsString string) (bool
 	); err != nil {
 		return false, err
 	}
-	return containAll(ns.Labels, r.namespaceLabels), nil
-}
 
-func containAll(labels map[string]string, labelsToCheck []string) bool {
```

Comment:
it's better to move this under a util package and add ut.

Reply:
Done